jenkins.jq

```diff
@@ -0,0 +1,18 @@
+# input: list of build objects i.e., builds.json
+# output: stream of crane copy command strings
+def crane_deploy_commands:
+	reduce (.[] | select(.build.resolved and .build.arch == env.BASHBREW_ARCH)) as $i ({};
```

Comment:
In the future, it might be worth splitting this select out to be the responsibility of the caller, but I don't think we should do that here :eyes:

(Especially since the use cases I have in mind aren't well formed yet, so it's not worth doing anything concrete yet)